README.md

```diff
@@ -21,12 +24,18 @@ without importing framework specific dependencies. Any framework built on Java [
 Add the Moesif dependency to your project's pom.xml file:
 
 ```xml
-<!-- Include jcenter repository if you don't already have it. -->
```

Comment:
@CodeYouMust this is a just reminder/check message if needed. Do you really want to remove it?

Reply:
@praves77 The binaries for this repository are available in Maven Central repository, which is usually enabled by default. Moesif no longer uses `jcenter` after it was [shutdown by JFrog](https://blog.gradle.org/jcenter-shutdown)
So this comment is now inaccurate.